test/common_nodes.hpp

```diff
@@ -0,0 +1,185 @@
+#ifndef TEST_COMMON_NODES
```

Comment:
to note @wirew0rm: this should be moved with the next refactoring to the regular blocklib.

For the time being, this is IMO better placed in `./test/blocklib/core/unit-test` ... this helps finding those block implementations that are of global/core interest.

Reply:
I'm not sure the same nodes are of interest to the global public and to tests. In tests, we will probably need strange nodes that nobody should use or write.

Reply:
still blocklib subdir is a better place than the qa root directory....